projects/risk-assessment-scores-to-delius/tech-docs/diag/risk-assessment-context-map.d2

```diff
@@ -0,0 +1,37 @@
+
+style { ...@../../../../script/style.style }
+vars { ...@../../../../script/style.vars }
+** { ...@../../../../script/style.all-style }
+
+context-oasys: OASys Context {
+  entities: Entities {
+    grid-columns: 3
+    pop: Person on Probation
+    rsr: RSR
+    osp: OSP
+    ogrs: OGRS score
+    date: Assessment date
+    event: Event number
+  }
+}
+
+context-delius: Delius Context {
+
+  interface: Risk Assessment\n and Delius {
+    type: Anti-Corruption\nLayer
+  }
+
+  entities: Entities {
+    grid-columns: 2
+    event: Event
+    mte: Management tier event
```

Comment:
Not an issue with the docs - but we should be able to remove the creation of these management tier event records now.  @mxco86 shall we put a story in for this?